packages/yargs/lib/webpack.js

```diff
@@ -1,11 +1,49 @@
 'use strict';
 
+const { EOL } = require('os');
+
 const prettyMS = require('pretty-ms');
+const prettyBytes = require('pretty-bytes');
+const chalk = require('chalk');
 
 const {
   internal: { BuildError, BuildWarning },
 } = require('@untool/webpack');
 
+const formatAssets = (assets) =>
+  assets
+    .filter(({ name }) => !name.endsWith('.map'))
+    .map(({ name, size, isOverSizeLimit }) => {
+      const prettySize = (() => {
+        if (isOverSizeLimit) {
+          if (chalk.enabled) {
+            return chalk.red(prettyBytes(size));
+          } else {
+            return `${prettyBytes(size)} !!!`;
+          }
+        } else {
+          return prettyBytes(size);
+        }
+      })();
+      return `${chalk.enabled ? chalk.gray('-') : '-'} ${name} (${prettySize})`;
```

Comment:
Would it make sense to align the columns? Maybe with `\t`? Also because we might have the chunk names in there too in a later iteration.

Reply:
Also is the ternary needed here? The content (`'-'`) is the same and `chalk` should figure out internally whether to apply the gray color or not.

Reply:
@ZauberNerd I tried using tabs but ultimately gave up as we are dealing with dynamic user provided strings (e.g. project name) here and we cannot know how many tabs we need to align everything. I also considered using something like [`cliui `](https://www.npmjs.com/package/cliui) or [`table`](https://www.npmjs.com/package/table), but ultimately decided to postpone it.